cmd/terramate/cli/cli_flags_helper.go

```diff
@@ -310,24 +305,28 @@ func formatFlag(haveShort bool, flag *kong.Flag) string {
 	flagString := ""
 	name := flag.Name
 	isBool := flag.IsBool()
-	isCounter := flag.IsCounter()
```

Comment:
these changes were not in #1965 but they were incompatible with v0.7.1 version of Kong, so O copied over the `formatFlag()` function from that version.